src/exchanges/gmocoin-client.js

```diff
@@ -0,0 +1,306 @@
+const BasicClient = require("../basic-client");
+const BasicMultiClient = require("../basic-multiclient");
+const { MarketObjectTypes } = require("../enums");
+const Ticker = require("../ticker");
+const Trade = require("../trade");
+const Level2Point = require("../level2-point");
+const Level2Snapshot = require("../level2-snapshot");
+const moment = require("moment");
+const { wait } = require("../util");
+
+class GMOCoinClient extends BasicMultiClient {
```

Comment:
So the big question is why are you using `BasicMultiClient`?  `BasicMultiClient` is required when you must create multiple sockets. I don't see anything in the documentation that indicates you are unable to reuse a single socket to subscribe to multiple channels.  I think you only need to use the `BasicClient` implementation below. If I'm wrong on that, can you please document the reasoning why `BasicMultiClient` is needed here.